config/action.d/badips.py

```diff
@@ -350,7 +350,7 @@ def update(self):
 				s = ips - self._bannedips
 				p = len(s)
 				self._banIPs(s)
-				self._logSys.info(
+				self._logSys.log(self.loglevel,
 					"Updated IPs for jail '%s' (-%d/+%d). Update again in %i seconds",
```

Comment:
@sebres, would have been interesting to keep `INFO` loglevel here.
So that we can have the individual _Banned IP_ messages in `DEBUG` level (so not logged if we use for example the global configuration `loglevel = INFO`), and this summary message in `INFO` level.
This was one of the purposes of this PR :)

Reply:
See #2245 for a quick fix. Thx !

Reply:
answered in PR